README.md

```diff
@@ -60,6 +60,11 @@ Amplication Data Service Generator is the component that generates the server-si
 
 To use this package as a library or as a CLI with other projects, follow the instructions on the package page.
 
+### [Amplication Scheduler](./packages/scheduler/README.md)
+
+Amplication Scheduler is a simple HTTP cron job scheduler.
```

Comment:
@iddan does this work for both HTTP and HTTPS requests? (Or is this a stupid question?...), and is it important to note HTTP? Because I was thinking of tweaking it to: Amplication Scheduler is a cron job scheduler that integrates with Docker and Docker Compose.

Reply:
HTTP as a transport protocol and not HTTP as the unsecured version of HTTPS - So, it can work both with HTTP and HTTPS but it is no important in this context